gyp/pylib/gyp/easy_xml.py

```diff
@@ -121,7 +122,10 @@ def WriteXmlIfChanged(content, path, encoding="utf-8", pretty=False, win32=False
 
     default_encoding = locale.getdefaultlocale()[1]
     if default_encoding and default_encoding.upper() != encoding.upper():
-        xml_string = xml_string.encode(encoding)
+        if (sys.version_info[0] >= 3) and (sys.version_info[1] >= 7):
+            xml_string = xml_string.encode(encoding)
+        else:
+            xml_string = xml_string.decode("cp1251").encode(encoding)
```

Comment:
This is a Windoze-only thing?

Reply:
Yes

Reply:
Then detect the OS and don't do it on Linux, macOS, and other real operating systems.